### DISCUSSION TITLE
This is actually a question, Nested scraping

### DISCUSSION THREAD

**Discussion by shellking4** [elixir-crawly/crawly#236] [General]
I have a scraper that is scraping data from a website. On the website we have products and each product has different colors options which come with a given set of images. My question is how to set up my scraper so that if I'm on a product page I can also check the other options pages in order to get the specific images. Sounds like a nested stuff but I cannot yet figure out how to implement it.  Thanks for any tips

**Comment by ivanhercaz** [comment]
Hi @shellking4! I can't answer your question now, because I would need to replicate what you tried, but I suggest you to move it to [Discussions](https://github.com/elixir-crawly/crawly/discussions) section and include more details about the HTML structure and what you have tried.

**Comment by Ziinc** [comment]
Performing specific html parsing and extraction is up to your own personal skill, this is outside of the scope of Crawly and crawling management, nor is it an issue with the library.
